samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/configuration/BaseWebSecurityConfig.java

```diff
@@ -78,9 +82,9 @@ public void configure(HttpSecurity http) throws Exception {
 
     http
         //
-        .authenticationProvider(this.authenticationProvider)
         // define all urls that are not to be secured
-        .authorizeRequests().antMatchers(unsecuredResources).permitAll().anyRequest().authenticated().and()
+        .userDetailsService(userDetailsService()).userDetailsService(inMemoryUserDetailsManager()).authorizeRequests()
```

Comment:
in the current version there is only one user details configured in this line. However if you debug the authentication process there are indeed two user details services involved. The custom one for granting the permissions while the other one (the in memory user details manager) for the password check.